app/schema_translation.py

```diff
@@ -0,0 +1,37 @@
+from babel.messages import pofile
+from app.utils import dumb_to_smart_quotes
+
+
+class SchemaTranslation:
+
+    catalog = []
+
+    def __init__(self, catalog=None):
+        self.catalog = catalog or self.catalog
+
+    def load(self, translation_file_path):
+        with open(translation_file_path, encoding='utf8') as translation_file:
+            self.catalog = pofile.read_po(translation_file)
+
+    def save(self, translation_file_path):
+        with open(translation_file_path, 'w+b') as translation_file:
+            pofile.write_po(translation_file, self.catalog)
+
+    def translate_message(self, message_to_translate):
+        for message in self.catalog:
+            if message.id and dumb_to_smart_quotes(message.id) == dumb_to_smart_quotes(message_to_translate['original']):
+                found = True
+                if message.auto_comments:
+                    first_comment = message.auto_comments[0]
+                    answer_id = first_comment.split(":")[1].strip()
+                    found = answer_id == message_to_translate['answer_id']
+
+                if found:
+                    return dumb_to_smart_quotes(message.string)
+
+    def translate(self, messages):
```

Comment:
I can only see this method being used in tests, or have I missed something?

Reply:
Yes, it was. Removed it now.